src/SFA.DAS.EmploymentCheck.Data/Repositories/EmploymentCheckRepository.cs

```diff
@@ -222,5 +222,58 @@ public async Task UpdateEmploymentCheckAsComplete(Models.EmploymentCheck check,
 
             await unitOfWork.ExecuteSqlAsync(sql, parameter);
         }
+
+        public async Task<long> ResetEmploymentChecksMessageSentDate(Guid correlationId)
+        {
+            var dbConnection = new DbConnection();
+            await using (var sqlConnection = await dbConnection.CreateSqlConnection(
+                _settings.DbConnectionString,
+                _azureServiceTokenProvider)
+            )
+            {
+                Guard.Against.Null(sqlConnection, nameof(sqlConnection));
+                await sqlConnection.OpenAsync();
+
+                var parameter = new DynamicParameters();
+                parameter.Add("@correlationId", correlationId, DbType.Guid);
+                parameter.Add("@lastUpdatedOn", DateTime.Now, DbType.DateTime);
+
+                const string sql = "UPDATE [Business].[EmploymentCheck] " +
+                                    "SET MessageSentDate = null, LastUpdatedOn = @lastUpdatedOn " +
+                                    "WHERE RequestCompletionStatus = 2 " +
+                                    "AND CorrelationId = @correlationId ";
+
+                var result = await sqlConnection.ExecuteAsync(sql, parameter, commandType: CommandType.Text);
+
+                return result;
```

Comment:
Unclear what the long type returned result means and what it's for. For commands there's no need to return anything.